.changes/fix-user-attention-minimized.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Fix: after minimizing the window by clicking the taskbar icon, the `requestUserAttention` function does not take effect.
```

Comment:
```suggestion
Fix `Window::request_user_attention` not taking effect after minimizing the window by clicking the taskbar icon

Reply:
Thanks for your suggestion, it has been modified